communication/moderation.md

```diff
@@ -1,38 +1,58 @@
 # Moderation on Kubernetes Communications Channels
 
+
 This page describes the rules and best practices for people chosen to moderate
 Kubernetes communications channels. This includes Github, Slack, forums, mailing
 lists, YouTube, Zoom, and any property listed in the SIG Contributor Experience
-[charter](/sig-contributor-experience/charter.md#code-binaries-and-services). 
+[charter].
 
-- Check the [centralized list of administrators] for contact information.
+- Check the [centralized list of administrators][moderators] for contact
+  information.
 - Some Kubernetes properties, like the Twitter account, are managed by the CNCF.
 
+---
+
+- [Selection of Moderators](#selection-of-moderators)
+  - [Moderators Pro Tempore](#moderators-pro-tempore)
+- [Rotation of Moderators](#rotation-of-moderators)
+- [Roles and Responsibilities](#roles-and-responsibilities)
+- [Escalation procedures](#escalation-procedures)
+- [Platform specific guidelines](platform-specific-guidelines)
+  - [Discuss](#discuss)
+  - [Mailing list](#mailing-list)
+  - [Slack](#slack-)
```

Comment:
nit: remove `-` or the link won’t work, LGTM otherwise

Reply:
Thanks for catching that! Completely overlooked. 👍